frame/elections-phragmen/src/lib.rs

```diff
@@ -197,7 +198,7 @@ pub mod pallet {
 	pub trait Config: frame_system::Config {
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
-		/// Identifier for the elections-phragmen pallet's lock
+		/// Identifier for the elections pallet's lock
```

Comment:
Also, pallet-id is used in locks, and if we change it, all locks need to be updated. This also needs to into the release notes.

Reply:
Added a note on this in the `CHANGELOG.md`